.travis.yml

```diff
@@ -11,10 +13,10 @@ install:
   - which node
   - clang++ -v
   - which clang++
-  - make distclean
```

Comment:
The author must have cleaned it up so it can be in line with other libraries
https://github.com/mapbox/vtvalidate/blob/master/.travis.yml#L11
https://github.com/mapbox/vtquery/blob/master/.travis.yml#L11